src/BingImageCreator.py

```diff
@@ -281,12 +283,14 @@ def __init__(
         debug_file: Union[str, None] = None,
         quiet: bool = False,
         all_cookies: List[Dict] = None,
+        proxy: str = None,
     ) -> None:
         if auth_cookie is None and not all_cookies:
             raise Exception("No auth cookie provided")
         self.session = httpx.AsyncClient(
             headers=HEADERS,
             trust_env=True,
+            proxies={"all://": proxy},
```

Comment:
its wrong here, if we o not have proxy it will fail 

use this way, refer this commit https://github.com/yihong0618/xiaogpt/commit/bce45ffdf5979a50c9f99862b926a68f78d2ed26

Reply:
The program ran fine locally without proxy, which is odd. Anyway, I followed your way, it has been changed.